### ISSUE TITLE
WebComponent support - specify where dynamic styles go instead of head

### ISSUE THREAD

**Issue by newlukai** [gridstack/gridstack.js#540]
We try to integrate Gridstack in a WebComponent/Polymer environment which means to embed Gridstack in a custom element. We recognized that the dynamically calculated stylesheets for the height of items is added to the `head` node, so the styles are not available inside the shadow DOM of the custom element.

In order to enable Gridstack for WebComponents it would be great if Gridstack would be aware of the Shadow DOM and for example allow to provide the DOM root where to place the dynamically created stylesheets.

**Comment by IftachSadeh** [comment]
Hey radiolips, 

Can you please say if this is expected to be fixed? Currently use with polymer is simply impossible. 
If it's not a priority, is there a recommended workaround in the meantime?

**Comment by VinodLouis** [comment]
Any one got workaround for this ?

**Comment by VinodLouis** [comment]
I tried something in case if anyone is interested https://github.com/VinodLouis/polymer-gridstack

**Comment by radiolips** [comment]
@VinodLouis Can you create a PR with changes such that gridstack would work as a web component?

**Comment by adumesny** [comment]
I use angular component for this lib, and having local to that components styles would be better and might be using webcomponents soon as well.

**Comment by adumesny** [comment]
the offending code is:
```js
var Utils = {
  createStylesheet: function(id) {
    var style = document.createElement('style');
    style.setAttribute('type', 'text/css');
    style.setAttribute('data-gs-style-id', id);
    if (style.styleSheet) {
      style.styleSheet.cssText = '';
    } else {
      style.appendChild(document.createTextNode(''));
    }
    document.getElementsByTagName('head')[0].appendChild(style);
    return style.sheet;
  },

  removeStylesheet: function(id) {
    $('STYLE[data-gs-style-id=' + id + ']').remove();
  },
}
```
which can be overridden using:
```js
$(function () {
  window.GridStackUI.Utils.createStylesheet = function(id) {
    ...new code...
  };

  $('.grid-stack').gridstack();
...
});
```

**Comment by adumesny** [comment]
sorry for taking so long to address. fixed in 0.6.4
